tests/unit/test_coco_format.py

```diff
@@ -81,6 +82,50 @@ def test_can_export_and_import_back(self):
                 back_dataset = Dataset.import_from(path_test_dir)
             compare_datasets(self, dataset, back_dataset)
 
+    @mark_requirement(Requirements.DATUM_GENERAL_REQ)
+    def test_can_import_from_any_cwd(self):
```

Comment:
Does this test include the 1) not specifying current directory case?

Reply:
Yes, `os.path.relpath` does return path without `.`.

```python
>>> import os
>>> cwd = os.getcwd()
>>> os.listdir()
['.git', '.github', '.gitmodules', 'explorer.png', 'LICENSE', 'NOTICE', 'datum.py', 'notebooks', 'pytest.ini', 'security.md', 'tests', '.markdownlint.yaml', 'ipas_default.config', 'build', 'test_progressive.py', 'pts_in_hull.npy', 'MANIFEST.in', 'README.md', 'CHANGELOG.md', 'requirements-default.txt', 'datumaro.egg-info', '.tox', 'media.py', '3rd-party.txt', '.pre-commit-config.yaml', 'contributing.md', '.pytest_cache', '.gitattributes', 'setup.py', 'requirements-dev.txt', 'colorization_deploy_v2.prototxt', 'docs', 'pyproject.toml', '.gitignore', 'datumaro', '.ruff_cache', 'cache', 'lightning_logs', 'logs', 'colorization_release_v2.caffemodel', '__pycache__', 'test.py', 'requirements.txt', 'tox.ini', 'requirements-core.txt']

>>> os.path.relpath(os.path.join(cwd, 'datumaro'), cwd)
'datumaro'
```